src/plugins/inspector/public/views/requests/components/details/req_code_viewer.tsx

```diff
@@ -119,38 +57,12 @@ export const RequestCodeViewer = ({
               </EuiCopy>
             </div>
           </EuiFlexItem>
-          {shouldShowDevToolsLink && (
-            <EuiFlexItem grow={false}>
-              <div>
-                <EuiButtonEmpty
-                  size="xs"
-                  flush="right"
-                  iconType="wrench"
-                  href={consoleHref}
-                  onClick={handleDevToolsLinkClick}
-                  data-test-subj="inspectorRequestOpenInConsoleButton"
-                >
-                  {openInConsoleLabel}
-                </EuiButtonEmpty>
-              </div>
-            </EuiFlexItem>
-          )}
-          {shouldShowsearchProfilerLink && (
-            <EuiFlexItem grow={false}>
-              <div>
-                <EuiButtonEmpty
-                  size="xs"
-                  flush="right"
-                  iconType="visBarHorizontal"
-                  href={searchProfilerHref}
-                  onClick={handleSearchProfilerLinkClick}
-                  data-test-subj="inspectorRequestOpenInSearchProfilerButton"
-                >
-                  {openInSearchProfilerLabel}
-                </EuiButtonEmpty>
-              </div>
-            </EuiFlexItem>
-          )}
+          {!!actions &&
+            actions.map((item) => (
+              <EuiFlexItem grow={false} key={item.name}>
```

Comment:
This looks much better. One thing  thing to improve would be to avoid using `index` as a key, https://medium.com/@sriweb/avoid-using-array-index-as-a-key-in-react-components-1978799ed94b

Reply:
I changed that.